internal/cli/kraft/lib/create/create.go

```diff
@@ -0,0 +1,315 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/erikgeiser/promptkit/textinput"
+	"github.com/spf13/cobra"
+
+	"kraftkit.sh/cmdfactory"
+	"kraftkit.sh/config"
+	"kraftkit.sh/packmanager"
+	"kraftkit.sh/unikraft/lib/template"
+)
+
+type CreateOptions struct {
+	ProjectName     string `long:"project-name" usage:"Set the project name to the template"`
+	LibraryName     string `long:"library-name" usage:"Set the library name to the template"`
+	LibraryKName    string `long:"library-kname" usage:"Set the library kname to the template"`
+	Version         string `long:"version" short:"v" usage:"Set the library version to the template"`
+	Description     string `long:"description"  usage:"Set the description to the template"`
+	AuthorName      string `long:"author-name" usage:"Set the author name to the template"`
+	AuthorEmail     string `long:"author-email" usage:"Set the author email to the template"`
+	InitialBranch   string `long:"initial-branch" usage:"Set the initial branch name to the template"`
+	CopyrightHolder string `long:"copyright-holder" usage:"Set the copyright holder name to the template"`
+	Origin          string `long:"origin" usage:"Source code origin URL."`
+	NoProvideMain   bool   `long:"no-provide-main" usage:"Do not provide provide-main to the template"`
+	GitInit         bool   `long:"git-init" usage:"Init git through the creating library"`
+	WithPatchedir   bool   `long:"patch-dir" usage:"provide patch directory to the template"`
+	SoftPack        bool   `long:"soft-pack" usage:"Softly pack the component so that it is available via kraft list"`
+	ProjectPath     string `long:"project-path" usage:"Where to create library"`
+}
+
+// Create creates a library template.
+func Create(ctx context.Context, opts *CreateOptions, args ...string) error {
+	if opts == nil {
+		opts = &CreateOptions{}
+	}
+
+	return opts.Run(ctx, args)
+}
+
+func NewCmd() *cobra.Command {
+	cmd, err := cmdfactory.New(&CreateOptions{}, cobra.Command{
+		Short:   "Initialise a library template",
```

Comment:
GitHub likes American formatting 😰 

```suggestion
		Short:   "Initialize a library from a template",
```

Reply:
Done 👍🏽